### PR TITLE
ORC-1330: Add `TestOrcConf`

### PR DESCRIPTION
**Pull request by deshanxiao**
### What changes were proposed in this pull request?
This PR is aimed to add a test for OrcConf.

### Why are the changes needed?
To Improve test coverage.

### How was this patch tested?
UT

### CODE REVIEW

**Comment by deshanxiao** [comment]
image

**Review by dongjoon-hyun — CHANGES_REQUESTED** [review]
Thank you for making a PR. Apache projects enforce to have LICENSE header. Could you add it to make CI happy?

File: java/core/src/test/org/apache/orc/TestOrcConf.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+import java.util.Properties;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TestOrcConf {
+
+    // Long
+    public final OrcConf ORC_STRIPE_SIZE_CONF = OrcConf.STRIPE_SIZE;
```
Comment (dongjoon-hyun) [inline java/core/src/test/org/apache/orc/TestOrcConf.java:15]: Use 2-space indentation please.

Comment (deshanxiao) [reply java/core/src/test/org/apache/orc/TestOrcConf.java]: Done

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

**Comment by dongjoon-hyun** [comment]
Merged to main/1.8. Thank you, @deshanxiao .

**Pull request closed without merging**